### ISSUE TITLE
unable to sign a did key doc with a did key...

### ISSUE THREAD

**Issue by OR13** [digitalbazaar/did-method-key-js#9]
```

  console.log tests/__fixtures__/documentLoader.js:23
    did:key:z6MkfBth5EfE8HgwtA9YfGgBCqqerTaeSKPPjy5aFHumngqj

  console.log tests/__fixtures__/documentLoader.js:23
    https://w3id.org/did/v1

 FAIL  tests/did-method-spec-conformance.spec.js
  did core spec > method conformance
    ✕ did:key (313ms)

  ● did core spec > method conformance › did:key

    The property "assertionMethod" in the input was not defined in the context.

      at Object.<anonymous>.module.exports.info (node_modules/jsonld-signatures/lib/expansionMap.js:9:11)
      at _expandObject (node_modules/jsonld/lib/expand.js:439:26)
      at Object.<anonymous>.api.expand (node_modules/jsonld/lib/expand.js:246:9)
```

https://w3id.org/did/v1 does not define assertionMethod, despite did:key relying on it.

**Comment by OR13** [comment]
Can be fixed easily by updating to `https://www.w3.org/ns/did/v1`

**Comment by OR13** [comment]
https://github.com/digitalbazaar/did-method-key-js/pull/10

**Comment by dmitrizagidulin** [comment]
Fixed in PR #10 / `v0.4.0` npm package. Closing.
